server/src/database/models/courseAssignment.ts

```diff
@@ -7,16 +7,17 @@ import {
 } from 'sequelize';
 
 import { sequelize } from '..';
-import CourseInstancePartialGrade from './courseInstancePartialGrade';
+import CourseInstance from './courseInstance';
 
 export default class CourseAssignment extends Model<
   InferAttributes<CourseAssignment>,
   InferCreationAttributes<CourseAssignment>
 > {
   declare id: CreationOptional<number>;
-  declare courseInstancePartialGradeId: ForeignKey<CourseInstancePartialGrade['id']>;
-  declare assignmentId: string;
-  declare maxPoints: number;
+  declare courseInstanceId: ForeignKey<CourseInstance['id']>;
+  declare name: string;
+  declare executionDate: Date;
```

Comment:
This name is a bit confusing, what exactly does this attribute represent?

Reply:
Added comment for clarification